src/e2e-test/features/zendesk/RunTime.feature

```diff
@@ -0,0 +1,231 @@
+    When Open Datafusion Project to configure pipeline
+    And Select data pipeline type as: "Data Pipeline - Batch"
+    And Select plugin: "Zendesk" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "Zendesk"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Enter input plugin property: "adminEmail" with value: "admin.email" for Credentials and Authorization related fields
+    And Enter input plugin property: "apiToken" with value: "admin.apitoken" for Credentials and Authorization related fields
+    And configure Zendesk plugin for listed subdomains:
+      | CLOUD_SUFI |
+    And Select dropdown plugin property: "objectsToPull" with option value: "Groups"
+    And Validate "Zendesk" plugin properties
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Connect source as "Zendesk" and sink as "BigQuery" to establish connection
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Configure BigQuery sink plugin for Dataset and Table
+    And Validate "Zendesk" plugin properties
+    And Close the Plugin Properties page
+    And Preview and run the pipeline
+    Then Verify the preview of pipeline is "success"
+
+  @TS-ZD-RNTM-02 @BQ_SINK
+  Scenario: Verify user should be able to deploy and Run the pipeline when plugin is configured for a Non hierarchical object
+    When Open Datafusion Project to configure pipeline
+    And Select data pipeline type as: "Data Pipeline - Batch"
+    And Select plugin: "Zendesk" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "Zendesk"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Enter input plugin property: "adminEmail" with value: "admin.email" for Credentials and Authorization related fields
+    And Enter input plugin property: "apiToken" with value: "admin.apitoken" for Credentials and Authorization related fields
+    And configure Zendesk plugin for listed subdomains:
+      | CLOUD_SUFI |
+    And Select dropdown plugin property: "objectsToPull" with option value: "Groups"
+    And Validate "Zendesk" plugin properties
+    And Close the Plugin Properties page
+    And Select Sink plugin: "BigQueryTable" from the plugins list
+    And Connect source as "Zendesk" and sink as "BigQuery" to establish connection
+    And Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    And Configure BigQuery sink plugin for Dataset and Table
+    And Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    And Save and Deploy Pipeline
+    And Run the Pipeline in Runtime
+    And Wait till pipeline is in running state
+    Then Verify the pipeline status is "Succeeded"
```

Comment:
For tests where we deploy and run a pipeline, please add a step to verify that the correct data was written to the sink.

Reply:
Added the step, Waiting for Github Actions PR to get merged first.